examples/java/pom.xml

```diff
@@ -273,7 +273,6 @@
       <artifactId>beam-runners-direct-java</artifactId>
       <version>${project.version}</version>
       <scope>runtime</scope>
-      <optional>true</optional>
```

Comment:
It's a cleanup. Since we remove dependency on beam-examples-java in other module, no longer need to set optional.